viofs/shared/fuse.h

```diff
@@ -175,15 +175,15 @@
  */
 
 // nonstandard extension used: zero-sized array in struct/union.
-#pragma warning(disable:4200)
```

Comment:
@YanVugenfirer this file is from the kernel, should we format it as a Windows file?

Reply:
Let's exclude it completely.